crates/m3/src/builder/table.rs

```diff
@@ -352,6 +352,7 @@ impl<'a, F: TowerField> TableBuilder<'a, F> {
 			namespaced_name,
 			ColumnDef::Constant {
 				poly: Arc::new(mle),
```

Comment:
Now that the data field is on here, the `poly` is probably unnecessary. It's fine to leave as is for now though.